.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you rebase your changes with the most recent change merged (run `git pull --rebase`)? Also, can you remove the .idea file from the commit and add the `.idea/` to the .gitignore file? It's something I've been meaning to do to avoid these files showing up in PRs